iphone/Maps/UI/Downloader/NoMaps/DownloaderNoResultsEmbedViewController.swift

```diff
@@ -0,0 +1,25 @@
+import UIKit
+
+@objc(MWMDownloaderNoResultsEmbedViewController)
+final class DownloaderNoResultsEmbed: UINavigationController {
+
+  @objc(MWMDownloaderNoResultsScreen)
+  enum Screen: Int {
+
+    case noMaps
+    case noSearchResults
+    
+  }
+
+  var screen = Screen.noMaps {
+    didSet {
+      let controller: MWMViewController
+      switch screen {
+      case .noMaps: controller = MWMNoMapsViewController.controller()
+      case .noSearchResults: controller = SearchNoResultsViewController.controller
+      }
+      setViewControllers([controller], animated: false)
+    }
+  }
+
```

Comment:
This is just like in enum - blank line after start & before end of class/struct/enum declaration.